tests/pitchfork_test.py

```diff
@@ -1,9 +1,54 @@
-pytest_plugins = ['errbot.backends.test']
-extra_plugin_dir = 'plugins'
+import github3
+import IGitt
+import plugins.labhub
+import plugins.pitchfork
 
+from unittest.mock import PropertyMock
+from unittest.mock import create_autospec
+from tests.corobo_test_case import CoroboTestCase
 
-def test(testbot):
-    testbot.assertCommand('!pitchfork @meet', 'being pitchforked')
-    testbot.assertCommand('!pitchfork @meet down to hell', 'being pitchforked')
-    testbot.assertCommand('!pitchfork meet to hell', 'being pitchforked')
-    testbot.assertCommand('!pitchfork', 'Usage')
+
+class TestPitchfork(CoroboTestCase):
```

Comment:
Many of our plugins which be changed to using labhub permissions.

create a LabHubTestCase subclassing CoroboTestCase which does all the setUp necessary for testing a plugin which depends on LabHub.